range_queries/SparseTable.cpp

```diff
@@ -30,21 +29,19 @@ void buildTable(int n) {
             if (curLen == 1)
                 table[i][j] = A[j];
             else
-                table[i][j] = min(table[i-1][j], table[i-1][j + curLen/2]);
+                table[i][j] = std::min(table[i-1][j], table[i-1][j + curLen/2]);
```

Comment:
`std::min` and `std::max` require `#include <algorithm>`

Reply:
It is compiling without any errors/warnings. Should I still include `#include<algorithm>`?

Reply:
yes please. it may compile on some platforms but not on others. if you can confirm it compiles on linux, Mac, Win using GCC, Clang and MS Visual C++ compilers in its current format, then I am ok :)

Reply:
you can check by creating a pull request [here](https://github.com/kvedala/C-Plus-Plus) which will compile on all these platforms for you and confirm

Reply:
Added and pushed final commit.